robotframework-ls/tests/robotframework_ls_tests/completions/test_dictionary_entries_completions.py

```diff
@@ -0,0 +1,68 @@
+def test_dictionary_entries_completions_1(
+    workspace, libspec_manager, data_regression
+):
+    from robotframework_ls.impl.completion_context import CompletionContext
+    from robotframework_ls.impl import dictionary_completions
+
+    workspace.set_root("case1", libspec_manager=libspec_manager)
+    doc = workspace.get_doc("case1.robot")
+    doc.source = """
+*** Variables ***
+&{Person}   First name=John   Last name=Smith
+
+*** Test Cases ***
+Dictionary Variable
+    Log to Console    ${Person}[First]"""
+    line, col = doc.get_last_line_col()
+    completions = dictionary_completions.complete(
+        CompletionContext(doc, workspace=workspace.ws, line=line, col=col - len("]"))
+    )
+    data_regression.check(completions)
```

Comment:
The actual files related to the data regression aren't added (so, tests will fail in the ci).